ajax/libs/jquery.basictable/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "jquery.basictable",
+  "version": "1.0.6",
+  "filename": "jquery.basictable.min.js",
+  "homepage": "https://www.jerrylow.com/basictable/demo/",
```

Comment:
Please use `https://www.jerrylow.com/basictable/`. 
With `demo`, it only shows demo. 
Without `demo`, users can see intro, setup, link to demo and so on.